neo.UnitTests/UT_PoolItem.cs

```diff
@@ -125,19 +115,10 @@ public Transaction GenerateTxWithFirstByteOfHashLessThanOrEqualTo(byte firstHash
         // Generate Transaction with different sizes and prices
         public static Transaction GenerateTx(long networkFee, int size, byte[] overrideScriptBytes = null)
         {
-            // use random bytes in the script to get a different hash since we cannot mock the Hash
-            byte[] randomBytes;
-            if (overrideScriptBytes != null)
-                randomBytes = overrideScriptBytes;
-            else
-            {
-                randomBytes = new byte[1];
-                TestRandom.NextBytes(randomBytes);
-            }
-
             Transaction tx = new Transaction
             {
-                Script = randomBytes,
+                Nonce = (uint)TestRandom.Next(),
```

Comment:
This Nonce used to come as a remark, right?

Reply:
What do you mean?

Reply:
Previously we append an Attribute for nonce behaviour

Reply:
`Nonce` is to generate different TXIDs. Now we don't have UTXO.